test/streaming.ts

```diff
@@ -38,6 +38,7 @@ import * as streaming from '../src/streaming';
 import {expect} from 'chai';
 import * as sinon from 'sinon';
 import * as through2 from 'through2';
+import {Stream, Duplex} from 'stream';
```

Comment:
Was this import needed? It wasn't in the initial PR changes.

Reply:
Dude your code review game is on point today 😆 Nice catch!